### PR TITLE
[FIX] website_sale: fix text color preview of ribbons

### PR DESCRIPTION
**Pull request by bvr-odoo**
Before this commit, the text color preview of the ribbons in the
ribbon selector was wrong. But also, if we selected another ribbon
just after changing the text color of a ribbon. This color was applied
to the selected ribbon.

task-2501515

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_sale/static/src/js/website_sale.editor.js [hunk 1]
```diff
@@ -448,6 +448,7 @@ options.registry.WebsiteSaleProductsItem = options.Class.extend({
      */
     createRibbon(previewMode, widgetValue, params) {
         this.saveMethod = 'create';
+        this.setRibbon(false);
```
Comment (rdeodoo) [inline addons/website_sale/static/src/js/website_sale.editor.js:451]: Isn't this enough? Tried locally and seems to work
```suggestion
        this._resetRibbonDummy();
```

Comment (bvr-odoo) [reply addons/website_sale/static/src/js/website_sale.editor.js:451]: @rdeodoo It doesnt work on my side.. I still have the same issue explained in the commit message

Comment (rdeodoo) [reply addons/website_sale/static/src/js/website_sale.editor.js:451]: Not sure how did I test it the other day, but it indeed does not work :shrug:

**Review by rdeodoo — APPROVED** [review]
LGTM, working as expected, preview and new style correctly applied/shown in all cases

**Comment by JKE-be** [comment]
@robodoo delegate=rdeodoo

**Comment by rdeodoo** [comment]
@robodoo r+

**Comment by JKE-be** [comment]
@robodoo rebase-ff

**Comment by rdeodoo** [comment]
@robodoo retry

**Pull request closed without merging**